package.json

```diff
@@ -2,14 +2,14 @@
   "name": "eslint-config-google",
   "version": "0.6.0",
   "description": "ESLint shareable config for the Google style",
-  "license": "Apache-2",
+  "license": "Apache-2.0",
   "repository": "google/eslint-config-google",
   "author": "Google",
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "ava"
+    "test": "node test/test.js"
```

Comment:
I was having problems running the tests with ava.

When executing the expression `new eslint.CLIEngine(eslintOpts).executeOnFiles(repoFiles);` via ava the expression would return different results than when executing it via node. I think this is because ava transpiles code prior to execution, and maybe that's breaking something.

I didn't take the time to debug it because it seemed fine to remove a dependency since I'm just testing one thing and `assert` works fine for that.